File: COM_SIC_Procesa_Pagos/DataPowerRest/ClaroFault.cs
using System;

namespace COM_SIC_Procesa_Pagos.DataPowerRest
{
	//INICIATIVA 712 Cobro Anticipado
	/// <summary>
	/// Summary description for ClaroFault.
	/// </summary>
	public class ClaroFault
	{
		public ClaroFault()
		{
			//
			// TODO: Add constructor logic here
			//
		}

		private string _idAudit;
		private string _codeError;
		private string _descriptionError;
		private string _locationError;
		private string _date;
		private string _originError;

		public string idAudit
		{
			set{_idAudit= value;}
			get{ return _idAudit;}
		}

		public string codeError
		{
			set{_codeError= value;}
			get{ return _codeError;}
		}

		public string descriptionError
		{
			set{_descriptionError= value;}
			get{ return _descriptionError;}
		}

		public string locationError
		{
			set{_locationError= value;}
			get{ return _locationError;}
		}

		public string date
		{
			set{_date= value;}
			get{ return _date;}
		}

		public string originError
		{
			set{_originError= value;}
			get{ return _originError;}
		}
		
	}
}

File: COM_SIC_Activaciones/ActualizarParticipante/ClaroFault.vb
Option Explicit On 

'INICIATIVA-219
Namespace ActualizarParticipante


    Public Class ClaroFault

        Private _idAudit As String
        Private _codeError As String
        Private _descriptionError As String
        Private _locationError As String
        Private _date As String
        Private _originError As String


        Public Property idAudit() As String
            Get
                Return _idAudit
            End Get
            Set(ByVal Value As String)
                _idAudit = Value
            End Set
        End Property

        Public Property codeError() As String
            Get
                Return _codeError
            End Get
            Set(ByVal Value As String)
                _codeError = Value
            End Set
        End Property

        Public Property descriptionError() As String
            Get
                Return _descriptionError
            End Get
            Set(ByVal Value As String)
                _descriptionError = Value
            End Set
        End Property

        Public Property locationError() As String
            Get
                Return _locationError
            End Get
            Set(ByVal Value As String)
                _locationError = Value
            End Set
        End Property

        'Public Property  Date() As String 
        '    Get
        '        Return _date
        '    End Get
        '    Set(ByVal Value As String)
        '        _date = Value
        '    End Set
        'End Property

        Public Property originError() As String
            Get
                Return _originError
            End Get
            Set(ByVal Value As String)
                _originError = Value
            End Set
        End Property



    End Class

End Namespace